spec/tools/fix_auth/fix_auth_spec.rb

```diff
@@ -1,7 +1,6 @@
 $LOAD_PATH << Rails.root.join("tools")
 
-require "fix_auth/fix_auth"
-require "fix_auth/models"
+require "fix_auth"
```

Comment:
This was a test fix exposed by running tests in parallel... ignore it, it will be in master soon